### ISSUE TITLE
About Page: sticky sidebar doesn't show full nav list when viewport's too short

### ISSUE THREAD

**Issue by shaunanoordin** [zooniverse/front-end-monorepo#5575]
## Minor UI Bug

Affects: `app-content-pages` (About -> Team and About -> Publications page)
Follows #5552 

The _newly-sticky_ sidebar on the About -> Team and About -> Publications page has an issue when the height of the browser viewport is shorter than the nav list.

- Expected: as a user viewing the page, I should be able to view all the items in the nav list, scrolling if necessary.
- Actual: the sticky nav list is "stuck" showing only the top of the list.

Screenshot: example of Publications page's nav list being shorter than the viewport. It only shows 8.5 items instead of 11, and attempting to scroll the nav list will instead scroll the content on the right.

<img width="853" alt="Screenshot 2023-11-01 at 17 28 15" src="https://github.com/zooniverse/front-end-monorepo/assets/13952701/94eae6fc-59dc-4371-9317-5bc6b534e9ce">

### Suggested Solution

For the StickySidebar component, adding `max-height: 100vh; overflow: auto` should allow users to scroll the nav list.

<img width="864" alt="Screenshot 2023-11-01 at 17 27 43" src="https://github.com/zooniverse/front-end-monorepo/assets/13952701/a228c78f-ed6f-4212-b2ba-a90b2ccb0eac">

### Status

Minor UI issue. I don't think anyone will realistically have a window shorter than the nav list, but fix should also be pretty simple, I think.

**Comment by eatyourgreens** [comment]
Worth testing this with very large fonts. It could be an accessibility issue when the page is zoomed in or the base font size is very large.

**Comment by shaunanoordin** [comment]
👍 I'll add _browser font size_ and _browser zoom_ to my manual test variables for my upcoming PR.

As a baseline, I tested b02ac8d63e43a17308268ef751922b581f5b0027 with font sizes and got the following on macOS + Chrome 118 (see Chrome settings at `chrome://settings/appearance`)

Set font size = medium (16px) and font size = very large (24px)
Result: guess which is which

<img width="400" alt="Screenshot 2023-11-01 at 18 03 03" src="https://github.com/zooniverse/front-end-monorepo/assets/13952701/2fe74fab-04c3-42d6-806b-4251ad9f3c2d">

<img width="400" alt="Screenshot 2023-11-01 at 18 04 41" src="https://github.com/zooniverse/front-end-monorepo/assets/13952701/bd456552-8ad0-4f79-af4b-5a4743e16c3b">

Set font size = very small (9px)
Result: oh, _now_ we see a change... but in the layout.

<img width="400" alt="Screenshot 2023-11-01 at 18 03 51" src="https://github.com/zooniverse/front-end-monorepo/assets/13952701/a672801f-78fe-48b3-8e03-e8eff35796ec">

Set MINIMUM font size = 24px (default 0px) via advanced font settings
Result: finally we see some changes

<img width="400" alt="Screenshot 2023-11-01 at 18 09 38" src="https://github.com/zooniverse/front-end-monorepo/assets/13952701/ebd20489-b64e-4fbc-a641-1d42f22f47ad">

**In summary:** atypical browser font sizes might be a blind spot for our CSS, since our page seems to ignore it. Or maybe it's a Grommet problem? (For comparison, setting font size = very large has a very obvious effect on a website like https://www.theguardian.com/ )

**Comment by eatyourgreens** [comment]
Out Grommet theme uses fixed pixel font sizes, which leads to the accessibility problem that you've described here: changing the browser's base text size won't resize fonts that are sized in absolute units.
